tests/qubit_codes/test_graphstates.py

```diff
@@ -38,38 +40,132 @@ def random_graph(request):
     return EGraph(G), G_adj, G_adj_sparse
 
 
+@pytest.fixture(scope="module", params=[N3D])
+def random_graph_3D(request):
+    """A convenience function to initialize random EGraphs for use in this
+    module."""
+    n = request.param
+    G = nx.Graph()
+    random_nodes = [
+        (random.randint(0, 100), random.randint(0, 100), random.randint(0, 100)) for _ in range(n)
```

Comment:
Need to make sure randomness is reproducible here (check out some of the other test files)

Reply:
Thanks for pointing this out! It's done ✅